docs/service_lifecycle.md

```diff
@@ -0,0 +1,62 @@
+# Service Lifecycle
+
+This defines the current service lifecycle stages, this isn't specific to the hosting platform
```

Comment:
full stop after stages. These are generic service definitions which are intended to clearly communicate the maturity of each service.